fix(spa): MonacoWrapper.test.tsx missing isActive prop — build broken on main

## Problem

\`pnpm --prefix spa run build\` fails on \`main\` (\`10100eb4\`):

\`\`\`
src/components/editor/MonacoWrapper.test.tsx(133,8): error TS2741: Property 'isActive' is missing in type '{ content: string; language: string; modelId: string; initialViewState: null; onChange: () => void; onCursorChange: () => void; onViewStateChange: () => void; onSave: Mock<Procedure>; }' but required in type 'Props'.
\`\`\`

## Root cause

\`MonacoWrapper\` requires prop \`isActive: boolean\` (\`src/components/editor/MonacoWrapper.tsx:9\`), but the rerender call site at \`MonacoWrapper.test.tsx:133\` was not updated when the prop was introduced.

Bisect points to PR #570 (\`f79e2c16\` / editor footer order) introducing / exposing the gap.

## Impact

- \`pnpm run build\` red on \`main\`. Blocks PR acceptance criteria "build green" for all feature branches downstream of \`10100eb4\`.
- Vitest does NOT catch this because the test runner is more permissive on missing props than \`tsc -b\`.
- HSR PR-3 (incoming) will carry this as a noted pre-existing blocker in its PR body.

## Fix

Add \`isActive={true}\` (or a contextually appropriate value) to the offending test instance at line 133. Likely also worth auditing neighbor test files for the same gap.

## Suggested prefix: \`fix(spa): restore build baseline\` — mirror PR #556 pattern.

--- a/spa/src/components/editor/MonacoWrapper.test.tsx
+++ b/spa/src/components/editor/MonacoWrapper.test.tsx
@@ -134,6 +134,7 @@ describe('MonacoWrapper', () => {
         content="hello"
         language="markdown"
         modelId="model-1"
+        isActive={true}
         initialViewState={null}
         onChange={() => {}}
         onCursorChange={() => {}}

